ftest.sh

```diff
@@ -395,6 +395,9 @@ if [[ \"${TEST_TAG_ARG}\" =~ soak ]]; then
     fi
 fi
 
+# DAOS-5622: Temporarily disable certs in CI
+INSECURE="--insecure_mode"
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]

Reply:
You do nothing... :P